src/components/ItemList.js

```diff
@@ -113,7 +116,13 @@ const ItemList = () => {
           <ul>
             {filteredResults
               ? filteredResults.map((item) => (
-                  <li key={item.id}>
+                  <li
+                    key={item.id}
+                    aria-label={`Need to buy ${item.data.name} ${itemStatus(
+                      item,
+                    )}`}
+                    className={itemStatus(item).replace(/\s+/g, '')}
```

Comment:
clever!

Reply:
We have fixed the aria-label for inactive items. It should now be more readable.

Reply:
Yep, the labels look more sensible now: 
![image](https://user-images.githubusercontent.com/2641172/155040762-db691679-c2d0-432d-aad0-86ab01e84633.png)